src/utils/daos.c

```diff
@@ -585,6 +585,11 @@ args_free(struct cmd_args_s *ap)
 	D_FREE(ap->user);
 	D_FREE(ap->group);
 	D_FREE(ap->principal);
+
+	if (ap->pool_label)
+		free(ap->pool_label);
+	if (ap->cont_label)
+		free(ap->cont_label);
```

Comment:
```suggestion
        D_FREE(ap->pool_label);
        D_FREE(ap->cont_label);
```

Reply:
as i mentioned, this can be used external to DAOS and we don't want to use those macros for tracing here.